tests/contracts-rust.spec.ts

```diff
@@ -61,49 +61,50 @@ beforeEach(
 );
 
 describe("Rust Smart Contracts", () => {
-  test("Flip contract", async (done): Promise<void> => {
-    const flipperAbi = require("../lib/ink/examples/lang/flipper/target/abi.json");
-    const STORAGE_KEY =
-      "0xeb72c87e65bed3596d6fef83aeb784615cdac1be1328adf1c7336acd6ba9ff77";
-    const abi: Abi = new Abi(flipperAbi);
-
-    // Deploy contract code on chain and retrieve the code hash
-    const codeHash: Hash = await putCode(
-      api,
-      testAccount,
-      "../lib/ink/examples/lang/flipper/target/flipper.wasm"
-    );
-    expect(codeHash).toBeDefined();
-
-    // Instantiate a new contract instance and retrieve the contracts address
-    const address: Address = await instantiate(
-      api,
-      testAccount,
-      codeHash,
-      abi.constructors[0],
-      CREATION_FEE
-    );
-    expect(address).toBeDefined();
-
-    const initialValue: Uint8Array = await getContractStorage(
-      api,
-      address,
-      STORAGE_KEY
-    );
-    expect(initialValue).toBeDefined();
-    expect(initialValue.toString()).toEqual("0x00");
-
-    await callContract(api, testAccount, address, abi.messages.flip());
-
-    const newValue = await getContractStorage(api, address, STORAGE_KEY);
-    expect(newValue.toString()).toEqual("0x01");
-
-    done();
-  });
+  // Currently broken, needs fixing after onk! 2.0 update
```

Comment:
```suggestion
  // Currently broken, needs fixing after ink! 2.0 update
```

Reply:
Let's ignore it instead of commenting?

Reply:
Just leave the broken test there...? (*itch)

I tried to get it to work yesterday for 2-3 hours and then just gave up on it for now because there were too many breaking changes in the last 2 weeks. I decided to let the dust settle and then try fixing this test again in 2-3 weeks or so

Reply:
Yes, I understand that the test is broken and we are not going to fix it right now.

I propose to mark it as ignored just because I find it to be a better way of conveying that. One reason for that, is that the test would still be type-checked which would prevent code rotting.

Reply:
Ah, ok, so you mean just `.skip` it?